x-pack/plugins/cases/common/constants.ts

```diff
@@ -149,3 +149,8 @@ export const CASES_TELEMETRY_TASK_NAME = 'cases-telemetry-task';
  */
 export const CASE_TELEMETRY_SAVED_OBJECT = 'cases-telemetry';
 export const CASE_TELEMETRY_SAVED_OBJECT_ID = 'cases-telemetry';
+
+export const enum CASE_VIEW_PAGE_TABS {
```

Comment:
Do we need these constants in the common folder? They seem too specific to the component.

Reply:
I believe later in the future we will want to deep-link things to specific tabs in the case view page. That's why I put them here.

Reply:
Deep links are not in this file either. I think it is not a good place to put them at the moment. If in the future we need them to be here we can always do it.

Reply:
OK I'll move them...